docs/developer-docs/release-procedure.md

```diff
@@ -0,0 +1,77 @@
+# Release Procedure
+
+This document is a Core team guide, provided publicly to ensure transparancy in the release process.
+
+## Versioning
+
+Jellyfin uses a 3-digit semantic versioning scheme. All releases will have versions in the `X.Y.Z` format, starting from `10.0.0`. Note however that the `10.Y.Z` release chain represents the "cleanup" of the codebase, so it should be accepted that `10.Y.Z` breaks all compatibility, at some point, with previous Emby-compatible interfaces.
+
+#### `X` - Major breaking versions
+
+The first digit represents a consistent release without breaking 3rd party interfaces including APIs.
```

Comment:
Can be more concise by having only bullet points for each section. Easier for ADHD folks like me to read. :P